README.md

```diff
@@ -27,5 +27,20 @@ Issues can be filed in the [issues section](https://github.com/azure/iotedge/iss
 ## Feature requests
 Feature requests can be filed on the [Azure IoT Edge User Voice page](https://feedback.azure.com/forums/907045-azure-iot-edge). 
 
-## Directory structure
-
+## Telemetry
+IoT Edge sends basic telemetry data to Microsoft about the host device. This data may be used to improve Microsoft products and services. This data includes:
+
+* CPU architecture
+* DMI System Manufacturer
```

Comment:
DMI?

Reply:
[Desktop Management Interface](https://en.wikipedia.org/wiki/Desktop_Management_Interface).  Perhaps too cryptic and should just be dropped in favor of simplicity?